### DISCUSSION TITLE
Red versus green outputs on a transaction?

### DISCUSSION THREAD

**Discussion by dwvisser** [mempool/mempool#1638] [General]
Looking at the following transaction details, how do I interpret the red versus the green arrows on the outputs?
![signal-2022-05-12-110111_001](https://user-images.githubusercontent.com/490575/168106473-e9707f3b-21de-42af-a3c6-d5c084894c12.png)

**Comment by dwvisser** [comment]
I think I found the answer to my own question, on https://www.blockchain.com/charts/output-volume, **emphasis mine:**

> The total value of all transaction outputs per day. **This includes coins returned to the sender as change.**

**Comment by softsimon** [comment]
Red means Spent UTXO and you can click on it to go to the transaction that spended it.

Green is Unspent and unclickable.
